examples/va/hci/control-plane/nncp/values.yaml

```diff
@@ -14,18 +14,21 @@ data:
     tenant_ip: 172.19.0.5
     ctlplane_ip: 192.168.122.10
     storage_ip: 172.18.0.5
+    swift_ip: 172.22.0.5
```

Comment:
I updated this value in my example, but when ci-framework deploys it's going to have to generate an example from what it deployed with this IP set. 

There's nothing in https://github.com/openstack-k8s-operators/ci-framework/pull/2301 that's doing that though.

Thus, it makes sense that `data.node_0.swift_ip` "is missing for replacement source"

Reply:
This change:

https://github.com/openstack-k8s-operators/ci-framework/pull/2301/files#diff-72446a72f1956b07c9bed327dca4e47da4f73d1b6e70f5c004cc44d1cb82e3fd

Made it green. Fixed.

https://logserver.rdoproject.org/86/386/7eaefc604a79f8058e13383159be65a554ab8d2e/github-check/rhoso-architecture-validate-hci/47b4d16/ci-framework-data/artifacts/ci_gen_kustomize_values/network-values/values.yaml